torchao/experimental/kernels/mps/test/test_lowbit.mm

```diff
@@ -200,22 +200,23 @@ void run_test(int32_t m, int32_t k, int32_t n, int32_t group_size) {
 
 template <typename T, int nbit>
 void run_test_battery() {
-  run_test<T, nbit>(1, 8, 1, 32);
-  run_test<T, nbit>(1, 32, 1, 32);
-  run_test<T, nbit>(1, 32, 1, 64);
-  run_test<T, nbit>(1, 56, 1, 64);
-  run_test<T, nbit>(1, 64, 1, 64);
-  run_test<T, nbit>(1, 72, 1, 64);
-  run_test<T, nbit>(1, 1000, 1, 64);
-  run_test<T, nbit>(3, 64, 5, 64);
-  run_test<T, nbit>(7, 64, 23, 64);
-  run_test<T, nbit>(17, 120, 23, 128);
-  run_test<T, nbit>(17, 128, 23, 128);
-  run_test<T, nbit>(41, 144, 23, 128);
-  run_test<T, nbit>(41, 128, 23, 128);
-  run_test<T, nbit>(81, 8, 1, 256);
-  run_test<T, nbit>(19, 256, 17, 256);
-  run_test<T, nbit>(1, 1000, 81, 256);
+  run_test<T, nbit>(1, 8, 4, 32);
```

Comment:
Not sure if this path is actually testing optimized kernels at all

Reply:
Ok so then my concern is that at any point in time you are only testing the kernel that the codegen is using via yaml struct. This is bit problematic to me. I dont know if I should hold this PR off to fix that but we definitely should fix it

Reply:
If you prefer, I can delete the shaders not used, until we have a mechanism to test of all of the shaders in the metal directory. Does that sound fair?